data/moves.ts

```diff
@@ -19437,13 +19437,20 @@ export const Moves: {[moveid: string]: MoveData} = {
 				}
 				return 5;
 			},
+			onModifyMove(move, source, target) {
+				if (!move.overrideOffensiveStat) return;
+				const statAndBoosts = move.overrideOffensiveStat;
+				if (!['def', 'spd'].includes(statAndBoosts)) return;
+				move.overrideOffensiveStat = statAndBoosts === 'def' ? 'spd' : 'def';
+				this.hint(`${move.name} uses ${statAndBoosts === 'def' ? '' : 'Sp. '}Def boosts when Wonder Room is active.`);
```

Comment:
This seems to have the same issue. `move.overrideOffensiveStat` is undefined for most moves, right? But Wonder Room should affect most moves, right?

Reply:
It will. This particular bit of code is there only to handle cases where the offensive stat is overwritten to a defensive stat. Attack stats are already swapped by the base wonderroom effect as coded in the stat fetching functions (I would LOVE to move that code onto wonderroom but it can't think of a good way to do it), but defense doesn't have this implementation so we need to add code to handle that edge case.

Reply:
Oh! I understand now.